build.gradle

```diff
@@ -84,7 +84,8 @@ repositories {
 spotless {
     java {
         target fileTree('.') {
-            include 'datasources/**/*.java',
+            include 'common/**/*.java'
```

Comment:
```suggestion
            include 'common/**/*.java',
```

Reply:
Thanks, I see you did it already in dee7452, thanks!